### ISSUE TITLE
Where should be retry and data fields?

### ISSUE THREAD

**Issue by Globik** [schwarzkopfb/sse-broadcast#4]
In options when initiating sse instance or direct in subscriber function? I d like to send data in setInterval.

**Comment by Globik** [comment]
Aha, I see in a publish function retry and pure data fields. But how to send data like setInterval?

**Comment by Globik** [comment]
I think I could like this:
// function publish(room, eventOrOptions, data, callback)
setInterval(function(){
Sse.publish('room',data:'hallo!')
},1000)
And then subsriber will every second send data to browsers)))

**Comment by schwarzkopfb** [comment]
If you need to control the retry filed per publish, you can pass a single object containing both retry and data fields:

```js
setInterval(function() {
  sse.publish('channel', { retry: 6000, data: 'hallo!' })
}, 1000)
```

But you can also specify a retry time when instantiating an `SSEBroadcaster`, and then that'll be used for all published message:

```js
const sse = require('sse-broadcast')({ retry: 6000 })

setInterval(function(){
  sse.publish('room', { data: 'hallo!' })
  // or simply
  sse.publish('room', 'hallo!')
},1000)
```

Do you have any other question or may I close this issue?

**Comment by Globik** [comment]
Yes, one can close this issue. I figured out how to. I did like this:
setInterval(function(){
sse.publish('ch_log_rooms',{data:"heart"});
},10000)

subrouter.get('/log_rooms', async (ctx,next)=>{
sse.subscribe('ch_log_rooms',ctx.res)
//console.log('header: ',ctx.request)
ctx.response=false;
await next();
})
I needed the same one channel for both: for a heartbeating and for a events.
Thank you for your reply and for a great packet!
